bedrock/newsletter/tests/test_views.py

```diff
@@ -513,10 +512,6 @@ def test_unknown_email(self, mock_basket):
         rsp = self.client.post(self.url, data)
         self.assertTrue(mock_basket.called)
         self.assertEqual(200, rsp.status_code)
-        form = rsp.context['form']
-        expected_error = unknown_address_text % \
-            reverse('newsletter.subscribe')
-        self.assertIn(expected_error, form.errors['email'])
```

Comment:
I wasn't sure how to test this with a Fluent string - is it still possible?

Reply:
I guess we could assert that the error text matches what we expect... or just call the `ftl()` here to get the text.